docs/types.rst

```diff
@@ -622,5 +622,6 @@ All type conversions in Vyper must be made explicitly using the built-in ``conve
 * Narrowing conversions (e.g., ``int256 -> int128``) check that the input is in bounds for the output type.
 * Converting between bytes and int types results in sign-extension if the output type is signed. For instance, converting ``0xff`` (``bytes1``) to ``int8`` returns ``-1``.
 * Converting between bytes and int types which have different sizes follows the rule of going through the closest integer type, first. For instance, ``bytes1 -> int16`` is like ``bytes1 -> int8 -> int16`` (signextend, then widen). ``uint8 -> bytes20`` is like ``uint8 -> uint160 -> bytes20`` (rotate left 12 bytes).
+* Enums can be converted to ``uint256`` only.
```

Comment:
What are the issues to converting to lesser values? Can't you determine the upper bound for a given Enum and throw a compiler or runtime error if it exceeds it?

Reply:
Since enums are `uint256` under the hood, I don't think there are any issues with supporting conversion to smaller integer types. It is more of whether we want to support that conversion.

Reply:
@fubuloubu i think the whole range of ints is a bit harder to test; we can add them later too